### PR TITLE
MED-20 Make open&hide button for Sidebar

### PR DESCRIPTION
**Pull request by pavlo-serhiichuk**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -15,10 +15,14 @@ import Profile from "./components/Profile/Profile.component";
 import WishesContainer from "./components/Wishes/WishesContainer";
 import {Routers} from "./AppContainer";
 import {BottomAnchor, GoBottom, GoTop, TopAnchor} from "./common/QuickScrolls/QuickScrolls.component";
+import {GiHamburgerMenu as Hamburger} from "react-icons/gi";
+import {LeftBtn} from "./components/Sidebar/Sidebar.style";
```
Comment (SVladikO) [inline src/App.js:21]: Don't import button style from other styles except this file name

File: src/App.js [hunk 2]
```diff
@@ -15,10 +15,14 @@ import Profile from "./components/Profile/Profile.component";
...
 
 const ProductsContainer = lazy(() => import("./components/Products/ProductsContainer"))
 
-function App({query, isSignInModalOpen, isLoginModalOpen, isMoreInfoModalOpen}) {
+function App(props) {
+    const {query, isSignInModalOpen, isSidebarOpen, open,
```
Comment (SVladikO) [inline src/App.js]: please put it into 23 row

File: src/App.js (hunk 1 above)
Comment (pavlo-serhiichuk) [reply src/App.js:21]: fixed

File: src/App.js (hunk 2 above)
Comment (pavlo-serhiichuk) [reply src/App.js]: fixed

**Pull request merged**